portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,52 @@
+.menu ul {
+    margin: 0;
+    padding: 0;
+    overflow: hidden;
+    list-style-type: none;
+}
+
+.menu ul li {
+    float: left;
+    padding: 0.5%;
+}
+
+.menu ul li:hover {
+    font-weight: bold;
+    border-bottom: 2px solid black;
+}
+
+.menu ul li a{
+    text-decoration: none;
+    text-align: center;
+    color: black;
+}
+
 #content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+    margin-left: auto;
```

Comment:
`#content` matches an element with an ID of `content`, but the div in your HTML that I think you are trying to address only has a class called `content`, so these rules aren't being applied; either the HTML or CSS should change.